### PR TITLE
Add SFS service

### PR DESCRIPTION
**Pull request by frisky-dis**
This PR adds a capability to manage OTC's SFS service.

### CODE REVIEW

**Comment by frisky-dis** [comment]
@niuzhenguo @freesky-edward  any updates on this?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 1.3.0 (Unreleased)
```
Comment (niuzhenguo) [inline CHANGELOG.md]: 1.2.0 is not released yet! Please add these change log to the section below.

File: opentelekomcloud/import_opentelekomcloud_sfs_file_system_v2_test.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package opentelekomcloud
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+// PASS
```
Comment (niuzhenguo) [inline opentelekomcloud/import_opentelekomcloud_sfs_file_system_v2_test.go]: no need to have such test result "//PASS"

File: opentelekomcloud/resource_opentelekomcloud_sfs_file_system_v2_test.go [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/huaweicloud/golangsdk/openstack/sfs/v2/shares"
+	"testing"
+)
+
+//PASS
```
Comment (niuzhenguo) [inline opentelekomcloud/resource_opentelekomcloud_sfs_file_system_v2_test.go]: ditto

File: opentelekomcloud/resource_opentelekomcloud_sfs_file_system_v2_test.go [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+						"opentelekomcloud_sfs_file_system_v2.sfs_1", "access_to", OS_VPC_ID),
+					resource.TestCheckResourceAttr(
+						"opentelekomcloud_sfs_file_system_v2.sfs_1", "access_type", "cert"),
+				),
+			},
+		},
+	})
+}
+
+// PASS
```
Comment (niuzhenguo) [inline opentelekomcloud/resource_opentelekomcloud_sfs_file_system_v2_test.go]: ditto

File: CHANGELOG.md (hunk 1 above)
Comment (Savasw) [reply CHANGELOG.md]: Done

File: opentelekomcloud/import_opentelekomcloud_sfs_file_system_v2_test.go (hunk 2 above)
Comment (Savasw) [reply opentelekomcloud/import_opentelekomcloud_sfs_file_system_v2_test.go]: Done

**Pull request merged**